gifted_build/postgres_manage.js

```diff
@@ -0,0 +1,97 @@
+const { Client } = require('pg');
+const knex = require('knex')({
+    client: 'pg',
+    connection: {
+      user: 'build',
+      host: 'localhost',
+      database: 'api',
+      password: 'password',
+      port: 5432
+    },
+    pool: { min: 0, max: 5 } // no logic to these numbers
+}); // Without this semicolon the program will break b/c Javascript has a silly parser.
+
+const product_images_table = "product_images"
+const products_table = "products"
+
+// Deletes existing tables and recreates them
+async function resetDB() {
+        // Reset the tables
+        console.log(`Dropping ${product_images_table} table`)
+        await knex.schema.dropTableIfExists(product_images_table)
+
+        console.log(`Dropping ${products_table} table`)
+        await knex.schema.dropTableIfExists(products_table)
+
+        console.log(`Creating ${products_table} table`)
+        await knex.schema.createTable(products_table, table => {
+            table.string('SKU', 255).primary() //SKUs should be unique
+            table.string('title', 255)
+            table.string('product_info', 511)
+            table.string('size_and_fit', 511)
+            table.string('materials_and_care', 511)
+            table.decimal('price', 6, 3) //6 digits precision (total number of digits), 5 digits scale (# of digits to right of decimal point)
+            // The product description displayed in bold to the right of the product image
+            table.string('description', 511)
+            // The "conventional" method is to use a linking table
```

Comment:
This definitely fine for our table as we they didn't specify about having multiple descriptions per item